actionview/lib/action_view/record_identifier.rb

```diff
@@ -86,11 +89,13 @@ def dom_class(record_or_class, prefix = nil)
     #
     #   dom_id(Post.find(45), :edit) # => "edit_post_45"
     #   dom_id(Post.new, :custom)    # => "custom_post"
-    def dom_id(record, prefix = nil)
+    #   dom_id(Post.find(45), :edit, :special) # => "edit_special_post_45"
+    #   dom_id(Post.new, :custom, :special)    # => "custom_special_post"
+    def dom_id(record, *prefixes)
       if record_id = record_key_for_dom_id(record)
-        "#{dom_class(record, prefix)}#{JOIN}#{record_id}"
+        dom_class(record, *prefixes, record_id)
```

Comment:
Ah sorry, that didn't work 😞 
The record_id should be added at the end but prefixes are added to the front. It should be reverted to:

```suggestion
        "#{dom_class(record, *prefixes)}#{JOIN}#{record_id}"

```

Or:
```suggestion
        [dom_class(record, *prefixes), record_id].join(JOIN)
```